src/Marketplace/MarketplaceScreen.js

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+// SETUP
+import React, { useEffect, useState } from 'react';
 import { View, StyleSheet } from 'react-native';
 import {
-  Title, Text, Button,
+  Title, Text, Subheading, Button, Card,
```

Comment:
Not sure if Subheading is actually used, if it isn't remove the subheading import